scipy/fft/_backend.py

```diff
@@ -86,6 +96,14 @@ def register_backend(backend):
     ------
     ValueError: If the backend does not implement ``numpy.scipy.fft``.
 
+    Examples
+    --------
+    We can register a new fft backend:
+
+    >>> from scipy.fft import fft, register_backend
+    >>> register_backend("scipy")  # Register a new backend
+    >>> fft([1])  # Calls the global backend first, and then use the new one
```

Comment:
In this case, the global backend will succeed and the registered version will not be called.

Reply:
Yes. But, I think there is no other backend which fails for calculation. It is difficult to call registered one. Do you have any idea?

Reply:
You could define one:
```python
class NoopBackend:
    __ua_domain__ = "numpy.scipy.fft"

    def __ua_function__(self, func, args, kwargs):
         return NotImplemented

scipy.fft.set_global_backend(NoopBackend())
```

That might not make for a good example though. The other option is change the comment to say the backend will only be called if the global backend returns `NotImplemented`.

Reply:
Oh. Thank you for your comment. I updated the docstring based on your comment. PTAL.